components/page.jsx

```diff
@@ -10,8 +10,23 @@ var DevRibbon = (process.env.NODE_ENV === 'production' &&
                  process.env.SHOW_DEV_RIBBON !== 'on')
                 ? null
                 : require('./dev-ribbon.jsx');
+var config = require('../lib/config');
 
 var Page = React.createClass({
+  statics: {
+    handlerForPage: function(router, url) {
+      return router.match(url).routes[1].handler;
```

Comment:
hey @toolness can you point me to the docs for `.match` method?  I could probably guess what that is for but I wanted to read through the docs to get a more clear sense.  I couldn't find it anywhere on https://github.com/rackt/react-router/tree/master/docs :confused: